tst/EnergyPlus/unit/UnitarySystem.unit.cc

```diff
@@ -188,6 +188,7 @@ class ZoneUnitarySysTest : public EnergyPlusFixture
         DataPlant::PlantLoop.allocate(DataPlant::TotNumLoops);
         DataSizing::PlantSizData.allocate(DataPlant::TotNumLoops);
         // int NumPltSizInput = DataPlant::TotNumLoops;
+        DataSizing::NumPltSizInput = 2;
```

Comment:
Added number of plant sizing object in the default constructor. The default constructor already has two plant loop sizing objects.